pkgs/data/themes/chili/plasma-chili.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "plasma-chili";
+  version = "0.5.5";
+
+  src = fetchFromGitHub {
+    owner = "MarianArlt";
+    repo = "kde-plasma-chili";
+    rev = "${version}";
+    sha256 = "17pkxpk4lfgm14yfwg6rw6zrkdpxilzv90s48s2hsicgl3vmyr3x";
+  };
+
```

Comment:
Missing qt dependencies.

Reply:
I don't think it needs any? They're not used directly by the theme, they need to be available wherever the theme is available (which I think is in SDDM). Or maybe they are already in `environment.systemPackages` from some leak somewhere else?

Reply:
I think I see what you're saying, I should test this with a system that has no other Qt desktop managers / libraries?

Reply:
Yeah like using SDDM without the KDE desktop environment.

Reply:
I guess you maybe need to do something like in https://github.com/NixOS/nixpkgs/pull/59016/files?

Reply:
Didn't have success with the `propagatedBuildInputs` since there seemed to be changes to the build process for qt. Settled for `propagatedUserEnvPkgs` on a local derivation.

Reply:
Or should it be using `qt.mkDerivation`? As is apparently now [the way](https://nixos.org/nixpkgs/manual/#sec-language-qt).

Reply:
@eadwu Can we merge without this being fixed?